contracts/clients/src/Aggregator.ts

```diff
@@ -1,3 +1,5 @@
+import { BigNumber, ContractReceipt } from "ethers";
+import { Log } from "@ethersproject/abstract-provider";
```

Comment:
CC - @jacque006 for using "fetch" outside browser

Reply:
Do we need to do something like `global.fetch = fetch` so that imported packages (like bls module) has access to fetch?  I had to do that in some scripts I've written

Reply:
Error in question was `Reference error: fetch is not defined`. Another workaround long-term might be to use node v18 which has the fetch API implemented by default.

https://stackoverflow.com/questions/48433783/referenceerror-fetch-is-not-defined

Reply:
We should support node versions without fetch until v16 rotates out of LTS support. I would recommend at some init point injecting `node-fetch` if it does not exist, esp if some other package is expecting it.

```
import fetch from "node-fetch";

if (!global.fetch) {
    global.fetch = fetch;
}
```